rockcraft.yaml

```diff
@@ -0,0 +1,163 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+---
+
+name: charmed-ranger-rock
+base: ubuntu@22.04
+version: 2.4.0-22.04-edge
+summary: Charmed ranger ROCK OCI
+description: |
+  Apache Ranger™ is a framework to enable, monitor and
+  manage comprehensive data security across the Hadoop platform.
+license: Apache-2.0
+
+platforms:
+  amd64:
+
```

Comment:
Having to pass stuff as environment variables for the image to start correctly is fine. There are plenty of cases of images that fail cause some env has not be passed, but that doesn't mean that there should not be an entry point. I think Kafka might be one of those cases: 
https://github.com/canonical/charmed-kafka-rock/blob/3-22.04/rockcraft.yaml
... in the server.properties it should have zookeeper configured, otherwise it can't start.

Do you think ranger works with sqlite?

Reply:
You're right, I have added these services. Not as far as I'm aware though based on experience just because there is zero documentation this could mean it's a fully supported feature of Ranger's :smile: